Runtime/Scripts/NetInterfaceIntegration.cs

```diff
@@ -93,39 +95,49 @@ public string GetIPAddress(string sourceNetInterfaceName, AddressFamily addressf
       return ipAddress;
     }
 
-    public bool HasCellular()
+    private bool testSameDefaultRoute(string iName)
     {
-      NetworkInterface[] netInterfaces = GetInterfaces();
-      bool foundByIp = false;
-
-      foreach (NetworkInterface iface in netInterfaces)
+      using (Socket socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, 0))
       {
-        foreach (string cellularName in networkInterfaceName.CELLULAR)
+        // Should not actually connect if UDP (connectionless)
```

Comment:
It's connecting what does that mean ?
<img width="237" alt="Screen Shot 2020-09-16 at 10 54 57 AM" src="https://user-images.githubusercontent.com/24863504/93374085-117db080-f80b-11ea-87db-ae944a54ed3c.png">

Reply:
Local endpoint, from resolved default socket network route to host.